pkg/apiserver/admission/initializer.go

```diff
@@ -38,36 +41,34 @@ type WantsInternalServiceCatalogInformerFactory interface {
 	admission.InitializationValidator
 }
 
-// WantsKubeClientSet defines a function which sets ClientSet for admission plugins that need it
-type WantsKubeClientSet interface {
```

Comment:
This change was most likely already made upstream, we just haven't adopted this change when rebased on the newer releases of Kubernetes.